client/backlog/backlog.html

```diff
@@ -0,0 +1,15 @@
+<head>
+<h1>Task List</h1>
+<form class="new-task">
```

Comment:
Let's not worry about the head at all. We have to deal with that in a special way. So pls remove this